src/app/company-page/company-page.component.css

```diff
@@ -1,3 +1,17 @@
-table{
-    width:100%;
-}
\ No newline at end of file
+table {
+    width: 1200px;
+}
+
+.demo-button {
```

Comment:
Would `add-data-button` be more intuitive for who reads the code? Or maybe `add-company-button`?

Reply:
Done